src/app/(site)/tickets/page.tsx

```diff
@@ -1,283 +1,71 @@
+      );
+
+      for (let i = 1; i <= numberOfSeats; i++) {
+        if (bookedSeatsArray.includes(i)) {
+          initialSeatsArray.push({ status: Seat.booked, seatNumber: i });
+        } else {
+          initialSeatsArray.push({ status: Seat.available, seatNumber: i });
+        }
+      }
+      setSeatsArray(initialSeatsArray);
+    };
+    fetchData();
+  }, []);
+
   const [ticketCount, setTicketCount] = useState(0);
+  const [seatsArray, setSeatsArray] = useState<
+    [] | { status: Seat; seatNumber: number }[]
+  >([]);
 
   return (
     <section className="sec-cont tickets-container">
       <h1>Biljettbokning</h1>
-      <div className="img-text-container">
-        <Image
-          src="/images/encanto.jpg"
-          alt="movie image"
-          width={141.75}
-          height={210}
-        ></Image>
-        <div>
-          <h2>Encanto</h2>
-          <p>Stora salongen idag, Onsdag 17.00</p>
-        </div>
-      </div>
-
-      <h2 className="header-two">Välj antal biljetter</h2>
-      <div className="number-of-tickets-container">
-        <div
-          onClick={() =>
-            setTicketCount(ticketCount != 0 ? ticketCount - 1 : ticketCount)
-          }
-          className="plus-minus"
-        >
-          -
-        </div>
-        <div className="ticket-count">{ticketCount}</div>
-        <div
-          onClick={() => setTicketCount(ticketCount + 1)}
-          className="plus-minus"
-        >
-          +
-        </div>
-      </div>
-
-      <h2 className="header-two">Välj platser</h2>
-
-      <div className="movie-screen-seat-container">
-        <div className="movie-screen-container">
-          <div className="movie-screen-left"></div>
-          <div className="movie-screen-middle"></div>
-          <div className="movie-screen-right"></div>
-        </div>
-
-        <div className="seat-container">
-          {MOCKSCREENING.map((seat) => {
-            return (
-              <div
-                key={seat.place}
-                className={
-                  seat.available ? 'seat-available' : 'seat-not-available'
-                }
-              >
-                {seat.place}
-              </div>
-            );
-          })}
-        </div>
-      </div>
-
-      <div className="choose-seat-container">
-        <div className="choosed-seat">☑️</div>
-        <p>Vald plats</p>
-      </div>
-      <div className="choose-seat-container">
-        <div className="booked-seat"></div>
-        <p>Upptagen</p>
-      </div>
-
-      <div className="divider"></div>
-
-      <div className="total-sum-container">
-        <h2>Att betala:</h2>
-        <h2>{ticketCount * 150}kr</h2>
-      </div>
-
-      <div className="divider"></div>
+      <TicketMovieInfo />
+      <TicketCount
+        ticketCount={ticketCount}
+        onClickMinus={() =>
+          setTicketCount(ticketCount != 0 ? ticketCount - 1 : ticketCount)
+        }
+        onClickPlus={() => setTicketCount(ticketCount + 1)}
```

Comment:
Add an if statement in the "onClickPlus" function that checks if you have reached the max tickets you can buy otherwise it might happen something like this.
![image](https://user-images.githubusercontent.com/32011023/237025273-66a6c543-6387-498e-afc4-fdfea1c3472f.png)